DAO.sol

```diff
@@ -500,6 +500,26 @@ contract DAO is DAOInterface, Token, TokenSale {
 
         uint quorum = p.yea + p.nay;
 
+        // require 53% for callin newContract
+        if (_transactionData.length >= 4 && _transactionData[0] == 0x68
+            && _transactionData[1] == 0x37 && _transactionData[2] == 0xff
```

Comment:
funny side effect is that if any other contract on the whitelist ever has the signature: `newContract(address)` it will fail here :)